dataland-backend/src/main/kotlin/org/dataland/datalandbackend/controller/DataController.kt

```diff
@@ -30,23 +32,38 @@ abstract class DataController<T>(
 
     override fun postCompanyAssociatedData(companyAssociatedData: CompanyAssociatedData<T>):
         ResponseEntity<DataMetaInformation> {
+        val userId = SecurityContextHolder.getContext().authentication.name
```

Comment:
We have a wrapper for that. I suppose using it.

Reply:
Guter Hinweis. Habe ich abgeändert.